common/lib/container-definitions/src/loader.ts

```diff
@@ -473,6 +473,9 @@ declare module "@fluidframework/core-interfaces" {
     }
 }
 
+/**
```

Comment:
can you add the version this is being deprecated from (59.1000 in this case) and add it to the BREAKING.md in a new upcoming changes section as well

Reply:
https://github.com/microsoft/FluidFramework/wiki/API-Deprecation